_config.yml

```diff
@@ -0,0 +1 @@
+
```

Comment:
yes, its a file in the sunpy.github.io in the website making repository. 
You can close this PR.
2 or 3 PRs has been collectively addressed here 
I will send separate PRs for the other related issues :)